lxd/network/driver_bridge.go

```diff
@@ -400,7 +400,7 @@ func (n *bridge) Validate(config map[string]string) error {
 	}
 
 	// Check IPv4 OVN ranges.
-	if config["ipv4.ovn.ranges"] != "" {
+	if config["ipv4.ovn.ranges"] != "" && shared.IsTrue(config["ipv4.dhcp"]) {
```

Comment:
ip{n}.dhcp defaults to enabled if not specified so we need to use `shared.IsTrueOrEmpty()` here and below.

Reply:
See https://linuxcontainers.org/lxd/docs/master/reference/network_bridge/

Reply:
Ahh I did wonder about that! Thanks.

Reply:
Fixed.